Add link to feedback form and tests

niqjohnson opened the pull request:
Adds link to the feedback form (and tests)
## Additions
- Link to feedback form from the "Tell us how" button at the bottom of the page
- Functional tests
## Changes
- The "Tell us how" `button` is now an `a`
## Testing
- To test, pull in the `feedback-link` branch and run `gulp`. Fire up the app in standalone or UnityBox as normal. The "Tell us how" button should open the feedback from in a new tab/window.
- I've tested this in IE8/Win7, IE9/Win7, IE10/Win7, IE11/Win7, Firefox/Win7, Chrome/Mac, Safari/Mac, Mobile Safari/iPhone6, and everything seems to be looking good
- Note that this branch is failing some unit and functional tests, just like `master` is right now. The changes here should add no additional failures.
## Review
- @marteki, @caheberer, @kurzn: I followed how the current tool works and am opening the feedback form in a new tab/window. Does that make sense, design-wise? (Actually, the current tool opens the form in a pop-up, which gets blocked by some browsers, so I'm using a plain old new tab/window with `target="_blank"`)
- @marteki, @mistergone: Do the new functional tests look OK? I added them in their own spec to prevent any merge conflicts with the other testing stuff that's being worked on; I can move them elsewhere if it makes more sense. Also, I'm pretty sure I'm testing for the presence of a new tab correctly, but let me know if that seems off to you.
- @higs4281: Is `href="/{{url_root}}/understanding-your-financial-aid-offer/feedback"` the Django-approved way to link to the feedback page?
## Checklist
- [X] Changes are limited to a single goal (no scope creep)
- [X] Code can be automatically merged (no conflicts)
- [X] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
- [ ] Passes all existing automated tests
- [X] Placeholder code is flagged
- [X] Visually tested in supported browsers and devices

niqjohnson committed:
Add link to feedback form and tests

higs4281 commented:
@niqjohnson not Django-approved, per se, but it allows us to switch from paying-for-college2 to the real thing by changing one view value rather than chasing down all the places it might be, when the time comes.

marteki commented:
Working in standalone Mac/Chrome and on my phone's Android Chrome Beta browser! I think the new tab interaction is the right choice.

I only saw one thing in the test spec to improve. I'll add the diff comment about it now. Keeping it in its own spec seems wise.

marteki commented on test/functional/settlementAidOfferPage.js:
```diff
@@ -425,6 +425,17 @@ settlementAidOfferPage.prototype = Object.create({}, {
+      }
+    },
+    followFeedbackLink: {
+      value: function( otherexpensesamount ) {
```
Can the `followFeedbackLink` function be similar to `confirmVerification` (line 23)? There's a unneeded parameter being passed, and it doesn't need to return anything.

niqjohnson committed:
Update tests

niqjohnson commented:
Updated `followFeedbackLink` to get rid of the extra stuff (I should probably look at what I'm copying an pasting before I copy and paste it...).

Also added a few lines to the tests to close the new tab that was opened in case more tests need to run in the original tab after.

marteki commented:
Looking and working snazzily, Niqorama! :+1: from me.

higs4281 commented:
:+1: works beautifully on unitybox, and the feedback is saved to the database. I had trouble on the last step on standalone, but that's not surprising and not a concern.
